.eslintrc

```diff
@@ -1,14 +1,12 @@
 {
-  "extends": [
-    "eslint:recommended",
-    "plugin:promise/recommended",
-    "prettier"
-  ],
-  "plugins": [
-    "promise"
-  ],
+  "parser": "babel-eslint",
```

Comment:
Can we rewrite this file to be an extension of `marketplace/.eslintrc`? Otherwise new rules we add won't be applied to this extension.

Even better, IMHO, would be if we could either have the code changed to conform to the rules or update `marketplace/.eslintrc` to have consistent code quality across the extensions in the `marketplace` directory.